interface/src/Application.cpp

```diff
@@ -381,12 +381,7 @@ const std::vector<std::pair<QString, Application::AcceptURLMethod>> Application:
 class DeadlockWatchdogThread : public QThread {
 public:
     static const unsigned long HEARTBEAT_UPDATE_INTERVAL_SECS = 1;
-    // TODO: go back to 2 min across the board, after figuring out the issues with mac
```

Comment:
hm I forget what this was for.  there might have been issues on startup?  or…shutdown?

Reply:
Whatever this was for, we don't know and the issue might not be a thing anymore. If there is a heartbeat taking over two minutes to complete, I feel like we should find out rather than just leaving it like this.